cert.go

```diff
@@ -111,9 +140,9 @@ func (m *mkcert) makeCert(hosts []string) {
 	}
 
 	if !m.pkcs12 {
-		log.Printf("\nThe certificate is at \"./%s.pem\" and the key at \"./%s-key.pem\" ✅\n\n", filename, filename)
+		log.Printf("\nThe certificate is at \"./%s\" and the key at \"./%s\" ✅\n\n", keyname, certname)
```

Comment:
Shouldn't the parameters be the other way around?

Reply:
Yes, thank you very much for this, i'll fix it right now.

Reply:
Great! Thanks for this PR - I hope it gets merged soon.